README.md

```diff
@@ -38,21 +38,45 @@ $ pip install requirements.txt
 
 ## Developers
 ```bash
+from ckan_sdk import f11s
 from ckan_sdk import client
 
-client_obj = client.Client(endpoint, token_api_key, organization_name, dataset_name)
-
-## Porcelain
-client_obj.push('path-to-resource')
-
-# you can also push a set of files
-# if directory we search for all *.csv ...
-client_obj.push_resources(resources)
-
-## Plumbing
-client_obj.store_blob()
-client_obj.push_resource_metadata() # resource_update
-client_obj.push_dataset_metadata()  # package_update
+client_obj = client.Client(endpoint, auth_token, organization_name)
+
+# loads a resource from a path
+resource = f11s.load(resource_file_path)
+# resource = {
+#     name: ...
+#     path: ...
+#     hash: ...
+#     size: ...
+#   }
+
+dataset = f11s.Dataset({'name': dataset_name})
+dataset.add_resource(resource)
+
+# Push the dataset and resources to CKAN and resources to cloud cloud
+res = client_obj.push(dataset.descriptor)
```

Comment:
Just wondering if it should be `push(dataset)` or `push(dataset.descriptor)`. Support either would be nice i guess ... (but that's a bonus).